test_rmw_implementation/test/test_client.cpp

```diff
@@ -149,6 +149,44 @@ TEST_F(CLASSNAME(TestClient, RMW_IMPLEMENTATION), create_with_bad_arguments) {
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
 }
 
+TEST_F(CLASSNAME(TestClient, RMW_IMPLEMENTATION), create_with_internal_errors) {
+  constexpr char service_name[] = "/test";
+  const rosidl_service_type_support_t * ts =
+    ROSIDL_GET_SRV_TYPE_SUPPORT(test_msgs, srv, BasicTypes);
+  RCUTILS_FAULT_INJECTION_TEST(
+  {
+    rmw_client_t * client =
+    rmw_create_client(node, ts, service_name, &rmw_qos_profile_default);
+    if (client) {
+      int64_t count = rcutils_fault_injection_get_count();
```

Comment:
Pausing the fault_injection happens a lot in these tests. If there isn't time to create a more obvious function call or macro, I would add a comment here so it's more clear why this is being done. Here and in the cases below.

Reply:
Alright, let's add it. I have to re-release `rmw_fastrtps` anyways, adding `rcutils` to the queue won't hurt.

Reply:
See https://github.com/ros2/rcutils/pull/295. I'll adapt this PR.